src/jsx.d.ts

```diff
@@ -1675,7 +1477,6 @@ export namespace JSXInternal {
 	interface DetailsHTMLAttributes<T extends EventTarget = HTMLDetailsElement>
 		extends HTMLAttributes<T> {
 		open?: Signalish<boolean | undefined>;
-		onToggle?: GenericEventHandler<T> | undefined;
```

Comment:
Luke saw this in #4694 -- essentially this type on a `GenericEventHandler` predates the better type we backported in #4615. Toggle events do trigger from popover targets now, so it's better as a global event type rather than a per-element type